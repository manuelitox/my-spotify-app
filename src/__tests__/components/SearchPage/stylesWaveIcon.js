import React from 'react'
import { shallow } from 'enzyme'

import StyledWaveSearchIcon from 'components/SearchPage/stylesWaveIcon'

describe('Component | StyledWaveSearchIcon', () => {
  let component 
  beforeEach( () => {
    component = shallow(
      <StyledWaveSearchIcon />
    )
  })
  it('should render a styled component', () => {
    expect(component).toMatchSnapshot()
  })
})