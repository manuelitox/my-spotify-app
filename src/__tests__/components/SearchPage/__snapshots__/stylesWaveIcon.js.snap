// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component | StyledWaveSearchIcon should render a styled component 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
  height: 60px;
  position: relative;
  top: -2px;
  width: 100%;
  @media only screen and (max-width: ",
          [Function],
          ") {
    height: 40px;
  }
",
        ],
      },
      "displayName": "Styled(ReactComponent)",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": [Function],
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
      Symbol(Symbol.hasInstance): [Function],
    }
  }
  forwardedRef={null}
/>
`;
